src/components/SquarePaymentForm.tsx

```diff
@@ -55,9 +55,9 @@ interface State {
  * This component requires 3 arguments for basic use:
  * * **applicationId**: This can be found in your [Square Developer Dashboard](https://connect.squareup.com/apps)
  * for the current Square app you're developing
- * * **locationID**: You can retrieve this from the [Square Connect v2 Locations API](https://docs.connect.squareupstaging.com/api/connect/v2#navsection-locations);
+ * * **locationID**: You can retrieve this from the [Square Connect v2 Locations API](https://docs.connect.squareup.com/api/connect/v2#navsection-locations);
  * or your [Square Developer Dashboard](https://connect.squareup.com/apps).
- * It determines which Square location will receive credit for transactions made with this form.
+ * It determines which Square location will receive credit for payments made with this form.
  * * **onCardNonceResponseReceived**: This callback gives you a nonce to pass to your back-end server to make a "charge" request to Square.
```

Comment:
Should this also be updated?

FYI the documentation for components is automatically generated from the comments (although it's not very clear at all). You can update the comment here, then run `./buildDocs.sh`. I haven't had a chance to fix the script yet so you will need to comment out lines 5-12 and line 72 of `SquarePaymentForm.tsx` before running the script.

Reply:
DPTW has been thinking of the proper use of the generic term "transaction",  In reality, payments and refunds are generically transactions but as you know, are represented by Payment and Refund.  Historically, we've used a Transaction to capture these transactions.  We want to reduce the chance of confusion ("Is Square talking about an transaction here or a Transaction?") so where the narrative describes a payment transaction (as in the payment form), we'll use "payment" instead of "transaction" so there's less ambiguity